cli/README.md

```diff
@@ -0,0 +1,213 @@
+# TRE CLI
+
+**WARNING - this CLI is currently experimental**
+
+## Installation
+
+Currently, I've only tested installation in the dev container. In this context, `make install-cli` should work
```

Comment:
Suggest rewording this to avoid `I've`
e.g. `Currently, installation has only been tested within the dev container.`

Reply:
👍